Documentation/network/ebpf/maps.rst

```diff
@@ -23,7 +23,7 @@ NAT                      node             512k            Max 512k NAT entries
 Neighbor Table           node             512k            Max 512k neighbor entries
 Endpoints                node             64k             Max 64k local endpoints + host IPs per node
 IP cache                 node             512k            Max 256k endpoints (IPv4+IPv6), max 512k endpoints (IPv4 or IPv6) across all clusters
-Load Balancer            node             64k             Max 64k cumulative backends across all services across all clusters
+Load Balancer            node             64k             Max ~3k clusterIP/nodePort Services (see: `service map tuning <#service-lb-map-tuning>`_ section for details).
```

Comment:
Just mentioning it, as I'm not sure it's considered in the PR. There's actually two maps involved in the LB decision:
* `LB4_SERVICES_MAP_V2`, which contains the frontends ("Virtual IPs"), and the associated backends
* `LB4_BACKEND_MAP`, which is indexed with the backendID and describes the backend

Also note-worthy is that entries in the backend map can be shared (== Services that use the same endpoints can re-use the backend value)

Reply:
good point, I was mostly focused on the frontends map as that seems to be the thing to fill up first in most cases.  The service LB is indexed on the backend slot/ID hence why estimating map size requirements is a bit trickier, as it would be per-service virtual IP.  It also may be worth noting that the average selected Pods heuristic may break down if you say, had one service of many that selected many thousands of Pods where this single service could dominate the svc LB map (maybe taking a max is actually safer?).

> Also note-worthy is that entries in the backend map can be shared.

Yeah, backends map could be described as being the limiting factor for number of backends. So it may be worth leaving the previous line in but renaming it to "Load Balancer Backends" or similar.

I think this one is easier to reason about, I think it's just limited by the total number of _unique_ selected Pod backends.

Reply:
@julianwiedmann  Update this to keep the old row, but rename it to `Load Balancer Backends`.  I also update the LB Services section to mention that this is across all clusters.